modules/messages/tests/server/message-to-influx.server.service.tests.js

```diff
@@ -0,0 +1,201 @@
+'use strict';
+
+/**
+ * Module dependencies.
+ */
+var mongoose = require('mongoose'),
+    path = require('path'),
+    async = require('async'),
+    User = mongoose.model('User'),
+    Message = mongoose.model('Message'),
+    messageToInfluxService =
+  require(path.resolve('./modules/messages/server/services/message-to-influx.server.service'));
+require('should');
+
+/**
+ * Unit tests
+ */
+describe('Message to influx server service Unit Tests:', function() {
+  var user1,
+      user2;
+
+  // here we create the users
```

Comment:
Not sure what we're doing elsewhere, but this type of database manipulation inside of tests leads to fragility. If the tests crash during execution the database may be left in an unclean state. I'd suggest that where possible it's better to use stubs and mocks.